storage/src/meta/toc.rs

```diff
@@ -335,56 +333,54 @@ impl TocEntryList {
         digest: &RafsDigest,
     ) -> Result<Self> {
         if !(512..=0x10000).contains(&size) || size % 128 != 0 {
-            return Err(anyhow!("invalid size {} of blob ToC", size));
+            return Err(eother!(format!("invalid size {} of blob ToC", size)));
         }
 
         let size = size as usize;
         let mut buf = alloc_buf(size);
         let sz = reader
             .read(&mut buf, offset)
-            .map_err(|e| anyhow!("failed to read data from backend, {:?}", e))?;
+            .map_err(|e| eother!(format!("failed to read data from backend, {:?}", e)))?;
 
         if sz != size {
-            return Err(anyhow!(
+            return Err(eother!(format!(
                 "failed to read data from backend, expect {}, got {} bytes",
-                size,
-                sz
-            ));
+                size, sz
+            )));
         }
 
         let dv = digest::RafsDigest::from_buf(&buf, digest::Algorithm::Sha256);
         if &dv != digest {
-            return Err(anyhow!("toc content digest value doesn't match"));
+            return Err(eother!("toc content digest value doesn't match"));
         }
 
         let size = size - 512;
         let header = Header::from_byte_slice(&buf[size..]);
         let entry_type = header.entry_type();
         if entry_type != EntryType::Regular {
-            return Err(anyhow!("Tar entry type for ToC is not a regular file"));
+            return Err(eother!("Tar entry type for ToC is not a regular file"));
         }
         let entry_size = header
             .entry_size()
-            .map_err(|_| anyhow!("failed to get entry size from tar header"))?;
+            .map_err(|_| eother!("failed to get entry size from tar header"))?;
         if entry_size != size as u64 {
-            return Err(anyhow!(
```

Comment:
Why not use `anyhow`

Reply:
It seems the `anyhow` crate only be introduced in `nydus-image`.